pythonbits/bb.py

```diff
@@ -8,9 +8,10 @@
 import shutil
 import re
 import subprocess
+import time
 
 from textwrap import dedent
-from collections import namedtuple
+from collections import namedtuple, abc
```

Comment:
Incompatible with Python 2, so drop Python 2 compat

Reply:
What exactly do you want me to do? If that line is incompatible with Python 2, Python 2 compat is already dropped.

Reply:
Ah, sorry for the misunderstanding, this is not meant for you to change, this is just a note on stuff that needs to be done after merging. So I want to merge it and then remove Python2 compatibility.